### PR TITLE
Correcting Powershell profile paths for Windows and Linux 

### PR DESCRIPTION
**Pull request by charub**
The Profiles were getting loaded from wrong paths. Correcting all profile paths.

### CODE REVIEW

**Comment by TravisEz13** [comment]
Charu, please update your GitHub profile

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs [hunk 1]
```diff
@@ -1722,10 +1722,13 @@ private void DoRunspaceInitialization(bool importSystemModules, bool skipProfile
                 // configuration to get the shell ID from, so we'll use the default...
                 string shellId = null;
                 if (_configuration != null)
-                    shellId = _configuration.ShellId;
+                        shellId = _configuration.ShellId;
```
Comment (zpallin) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs]: I don't think the added tab here was necessary

Comment (TravisEz13) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs]: I don't see a tab when I look at the file.  May be an issue with GitHub diff.

---
In reply to: [75389938](https://github.com/PowerShell/PowerShell/pull/1845#discussion_r75389938) [](ancestors = 75389938)

**Comment by daxian-dbw** [comment]
@charub this is a simple fix that doesn't seem to need two commits. Could you please squash them into one commit?

**Comment by vors** [comment]
@daxian-dbw you can do that yourself while merging. I think we settled to do that according to @lzybkr

**Comment by daxian-dbw** [comment]
@vors that comment was made before we enabled the squash feature. Now we are waiting for Charu to address @mirichmo 's comment.

**Comment by charub** [comment]
Suppressed by #2155

**Pull request closed without merging**